android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/BluetoothSite.java

```diff
@@ -228,7 +228,16 @@ private File getHtmlFile() {
     return new File(websiteDir, "website.html");
   }
 
+  private File getTempFile() {
+    File websiteDir = new File(activity.getFilesDir(), "Websites");
+    websiteDir.mkdir();
+    return new File(websiteDir, "temp.html");
```

Comment:
See File.createTempFile() http://stackoverflow.com/questions/3425906/creating-temporary-files-in-android